artifactory.py

```diff
@@ -2084,6 +2082,163 @@ def symlink_to(self, target, target_is_directory=False):
         raise NotImplementedError()
 
 
+class ArtifactoryBuild(ArtifactoryPath):
+    def __new__(cls, *args, **kwargs):
+        """
+        pathlib.Path overrides __new__ in order to create objects
+        of different classes based on platform. This magic prevents
+        us from adding an 'auth' argument to the constructor.
+        So we have to first construct ArtifactoryPath by Pathlib and
+        only then add auth information.
+        """
+        obj = super().__new__(cls, *args, **kwargs)
+        obj.project = kwargs.get("project", "")
+        return obj
+
+    @property
+    def builds(self):
+        """
+        Get all available builds on Artifactory
+        :return: (list) list of available build names
+        """
+        all_builds = []
+        url = ""
+        if self.project:
+            url = f"?project='{self.project}'"
```

Comment:
Let's remove python3.5 support, it's out of date.

Otherwise it gives an error about f-string in CI https://app.travis-ci.com/github/devopshq/artifactory/jobs/522657163

Reply:
done